Bio/GenBank/Scanner.py

```diff
@@ -505,6 +509,7 @@ def parse_cds_features(self, handle,
             self.parse_header()  # ignore header lines!
             feature_tuples = self.parse_features()
             # self.parse_footer() # ignore footer lines!
+            line = ""
```

Comment:
Yep, the variable was defined in a logic block and then used outside of that block. I found a couple of these.